pandas/core/arrays/period.py

```diff
@@ -641,11 +641,14 @@ def _format_native_types(
         """
         values = self.astype(object)
 
+        # Create the formatter function
         if date_format:
             formatter = lambda per: per.strftime(date_format)
         else:
+            # Uses `_Period.str>format_period`
```

Comment:
does the ">" mean something like "which in turn uses"?

Reply:
indeed. Do you prefer something more explicit ?

Reply:
yeah pls spell things out